### PR TITLE
Create Assessment

### PR DESCRIPTION
**Pull request by J14P**
Working code to create the assessment, collect the data from it, and pass it into the database

### CODE REVIEW

File: packages/api/models/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
+
+fs
+  .readdirSync(__dirname)
```
Comment (isa4ac) [inline packages/api/models/index.js:20]: Not sure what this is, but is it required?

File: packages/client/src/components/questions.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
+/* eslint-disable max-len */
+import React, { Component } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const options1 = [
```
Comment (isa4ac) [inline packages/client/src/components/questions.jsx:5]: Should this be in the database?

File: packages/client/src/components/questions.jsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+  render() {
+    const { value, value2, value3, value4, value5 } = this.state;
+    return (
+      <>
+        <h2>Questions & Responses</h2>
+        <ol>
+          <li><p>Previous contact with the Cat Judicial System</p>
+            <ul>
+              <li><label><input type="radio" value="0" name="value" checked={value === options1.value} onChange={() => this.setState({ selectedOption: options1.value })} /> No (score = 0)</label></li>
```
Comment (isa4ac) [inline packages/client/src/components/questions.jsx:49]: Should you be using bootstrap?

File: packages/api/models/index.js (hunk 1 above)
Comment (J14P) [reply packages/api/models/index.js:20]: i don't think I did anything with this lol

File: packages/client/src/components/questions.jsx (hunk 2 above)
Comment (J14P) [reply packages/client/src/components/questions.jsx:5]: it probably could be stored in the database, but it's hard coded for this assignment

File: packages/client/src/components/questions.jsx (hunk 3 above)
Comment (J14P) [reply packages/client/src/components/questions.jsx:49]: I still need to implement bootstrap

**Pull request merged**